src/stratis_cli/_actions/_data.py

```diff
@@ -256,6 +268,11 @@ def _get_timeout(value):
         environ.get("STRATIS_DBUS_TIMEOUT", DBUS_TIMEOUT_SECONDS * 1000)
     )
 
+    fetch_properties_spec = ET.fromstring(SPECS[FETCH_PROPERTIES_INTERFACE])
+    FetchProperties = make_class("FetchProperties", fetch_properties_spec, timeout)
+    MOFetchProperties = managed_object_class("MOFetchProperties", fetch_properties_spec)
```

Comment:
Omit these two lines, they're only used for getting objects that have properties out of ```GetManagedObjects``` result, so I don't think we will be using the generated classes ever.

Reply:
Resolved.